app/controllers/admin_controller.rb

```diff
@@ -61,6 +61,85 @@ def audit_log
+    settings.find_by(name: 'SiteName').update(value: @new_community.name)
+
+    # Audit Log
+    AuditLog.admin_audit(event_type: 'new_site', related: @new_community, user: current_user,
+                         comment: "<<Community #{@new_community.attributes_print}>>")
+
+    # Clear cache
+    Rails.cache.clear
+
+    # Render template
+    render
+  end
+
+  def setup; end
+
+  def setup_save
+    settings = SiteSetting.for_community_id(@community.id)
+    default_settings = SiteSetting.for_community_id(Community.first.id)
+
+    # Set settings from config page
+    settings.find_by(name: 'SiteCategoryHeaderDefaultColor').update(value: params[:primary_color])
+    settings.find_by(name: 'SiteLogoPath').update(value: params[:logo_url])
+    settings.find_by(name: 'SiteAdSlogan').update(value: params[:ad_slogan])
+    settings.find_by(name: 'MathJaxEnabled').update(value: params[:mathjax])
+    settings.find_by(name: 'SyntaxHighlightingEnabled').update(value: params[:syntax_highlighting])
+    settings.find_by(name: 'ChatLink').update(value: params[:chat_link])
+    settings.find_by(name: 'AnalyticsURL').update(value: params[:analytics_url])
+    settings.find_by(name: 'AnalyticsSiteId').update(value: params[:analytics_id])
+    settings.find_by(name: 'AllowContentTransfer').update(value: params[:content_transfer])
+
+    # Auto-load settings
+    settings.find_by(name: 'AdminBadgeCharacter')
+            .update(value: default_settings.find_by(name: 'AdminBadgeCharacter').value)
+    settings.find_by(name: 'ModBadgeCharacter')
+            .update(value: default_settings.find_by(name: 'ModBadgeCharacter').value)
+    settings.find_by(name: 'SEApiClientId')
+            .update(value: default_settings.find_by(name: 'SEApiClientId').value)
+    settings.find_by(name: 'SEApiClientSecret')
+            .update(value: default_settings.find_by(name: 'SEApiClientSecret').value)
+    settings.find_by(name: 'SEApiKey')
+            .update(value: default_settings.find_by(name: 'SEApiKey').value)
+    settings.find_by(name: 'AdministratorContactEmail')
+            .update(value: default_settings.find_by(name: 'AdministratorContactEmail').value)
+
+    # Generate meta tags
+    tags = [
+      'discussion', 'support', 'feature-request', 'bug',
+      'status-completed', 'status-declined', 'status-review', 'status-planned', 'status-deferred'
+    ]
+    Tag.create(tags.map { |t| { name: t, community_id: @community.id, tag_set: TagSet.meta } })
+
+    # Set Meta tags as required/mod-only
+    meta_category = Category.where(name: 'Meta').last
+    meta_category.required_tags << Tag.where(name: ['discussion', 'support', 'feature-request', 'bug'])
+    meta_category.moderator_tags << Tag.where(name: ['status-completed', 'status-declined', 'status-review',
+                                                     'status-planned', 'status-deferred'])
+
+    # Clear cache
+    Rails.cache.clear
+
+    # Audit Log
```

Comment:
**suggestion** (non-blocking): unnecessary comment - pretty clear what the following line does already